src/components/presentational/TimeSeriesChart/TimeSeriesChart.tsx

```diff
@@ -129,10 +159,36 @@ export default function TimeSeriesChart(props: TimeSeriesChartProps) {
                 } else if (domainMin !== undefined) {
                     domain = [domainMin, "auto"];
                 }
+
+                if (isNumber(domainMin)) nDomainMin = domainMin;
+                let domainMax = (props.options as MultiSeriesLineChartOptions).domainMax ?? 'auto';
+                if (isNumber(domainMax)) nDomainMax = domainMax;
             }
         }
 
         return <>
+            <defs>
```

Comment:
Instead of having these defs in the standard chart components, can you move them down into the line chart code similar to how it is done with bar charts?

Also, I'm not sure I understand the significance of the `lineKeyPrefix`.  Can't the line chart threshold gradients just always use the `gradientKey` like the bar chart ones do?

Reply:
@greinard Good feedback, I cleaned this up